dev/com.ibm.ws.security.mp.jwt_fat_tck/fat/resources/permissions.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2018 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<permissions xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
+        http://xmlns.jcp.org/xml/ns/javaee/permissions_7.xsd"
+    version="7">
+   
+   <permission>
+       <class-name>java.lang.RuntimePermission</class-name>
+       <name>getClassLoader</name>
+   </permission> 
+       
+   <permission>
+       <class-name>javax.security.auth.PrivateCredentialPermission</class-name>
+       <name>* * "*"</name>
```

Comment:
Is this a valid permissions name? I would've expected just `<name>*</name>`.

Reply:
yep that strange syntax is actually required.  Copied from security manager tests.

Reply:
actually I'm going to cancel this request and replace it with another that doesn't have a merge conflict